packages/terra-navigation-side-menu/tests/wdio/navigation-side-menu-spec.js

```diff
@@ -0,0 +1,41 @@
+/* global browser, Terra, before */
+
+describe('Default navigation side menu display', () => {
+  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+  beforeEach(() => {
+    browser.url('/#/raw/tests/navigation-side-menu/navigation-side-menu-default');
+    browser.waitForVisible('#test-menu');
+    browser.moveToObject('#test-menu #test-item-1');
```

Comment:
I mentioned this, but IMO we should break out this hover into its own test step.

Reply:
Updated.